### ISSUE TITLE
中文问题

### ISSUE THREAD

**Issue by xuzhendan** [vy/log4j2-redis-appender#6]
写入中文redis不直接读取中文，可以处理下吗

**Comment by vy** [comment]
Hey @xuzhendan! I've consulted to a Chinese friend and Google Translate, but neither managed to make sense of your question. Would you mind elaborating it (**in English!**) a bit, please? Some details on how to reproduce the problem locally will be really helpful.

**Comment by xuzhendan** [comment]
![image](https://user-images.githubusercontent.com/6902752/62018635-1ab65300-b1ee-11e9-81fe-bd449ad54715.png)
I wan to show Chinese, in redis console.

**Comment by vy** [comment]
@xuzhendan, a Log4j `Appender` has no control over the serialization of the `LogEvent`. You pass it a `Layout` (e.g., `PatternLayout`) and the `Appender` uses `byte[] Layout#toByteArray(LogEvent)` method to serialize the `LogEvent`. (See [RedisAppender.java#L138](https://github.com/vy/log4j2-redis-appender/blob/master/appender/src/main/java/com/vlkan/log4j2/redis/appender/RedisAppender.java#L138).)

Long story short, I suspect if your `Layout` is configured right. It looks like you are using the [`PatternLayout`](https://logging.apache.org/log4j/2.0/manual/layouts.html#PatternLayout). If so, did you configure its `charset` property right? If not, which appender are you using?

**Comment by vy** [comment]
Closing the ticket due to insufficient information and reproduction path.
